docs/src/components/navigation/social-links.tsx

```diff
@@ -0,0 +1,83 @@
+import Link from 'next/link';
+
+export function SocialLinks({ tabIndex }: { tabIndex?: number }) {
+  const styles =
+    'shrink-0 rounded-lg p-3 hover:bg-keystatic-gray-light active:bg-keystatic-gray transition-colors';
+
+  return (
+    <div className="flex gap-2">
+      <Link
+        href="https://fosstodon.org/@keystatic"
+        className={styles}
+        tabIndex={tabIndex}
+      >
+        <span className="sr-only">Keystatic on Mastodon</span>
```

Comment:
`sr-only` - neat!

Reply:
Can't take the credit for this one unfortunately :D